--- empires-server_OG.py	2020-05-02 22:56:47.000000000 +0700
+++ empires-server_COMPUTERNAMECHANGE.py	2020-05-10 08:15:29.414147300 +0700
@@ -1501,7 +1501,7 @@
 
     if step == "setState":
         if lookup_object(id)["referenceItem"] == None and reference_item != None:
-            costs = lookup_item_by_code(reference_item).get("cost")
+            costs = lookup_item_by_code(reference_item.split(":")[0]).get("cost")
             do_costs({k: v for k, v in costs.items() if k != "-cash"})
         lookup_object(id)["referenceItem"] = reference_item
 
