--- empires-server_singlethreaded.py	2020-05-01 05:06:42.998454700 +0700
+++ empires-server_multithreaded.py	2020-05-01 05:06:39.037011300 +0700
@@ -2078,7 +2078,7 @@
     # app.session_interface.db.create_all()
     # db.create_all()

-    socketio.run(app, host=host, port=port, debug=debug)
+    socketio.run(app, threaded="True", host=host, port=port, debug=debug)
     # app.run(host='127.0.0.1', port=5005, debug=True)
     # logging.getLogger('socketio').setLevel(logging.ERROR)
     # logging.getLogger('engineio').setLevel(logging.ERROR)
